python/lancedb/query.py

```diff
@@ -145,3 +145,26 @@ def to_df(self) -> pd.DataFrame:
             },
         )
         return tbl.to_pandas()
+
+
+class LanceFtsQueryBuilder(LanceQueryBuilder):
+    def to_df(self) -> pd.DataFrame:
+        try:
+            import tantivy
+        except ImportError:
+            raise ImportError(
+                "You need to install the `lancedb[fts]` extra to use this method."
+            )
+
+        from .fts import search_index
+
+        # get the index path
+        index_path = self._table._get_fts_index_path()
+        # open the index
+        index = tantivy.Index.open(index_path)
```

Comment:
this does not support multi-column?

Reply:
So with fts indices you can have a single index that covers multiple columns.